tools/demikernel_ci.py

```diff
@@ -372,9 +351,13 @@ def main():
     server_addr: str = args.server_addr if test_system else ""
     client_addr: str = args.client_addr if test_system else ""
 
-    status: int = run_pipeline(repository, branch, libos, is_debug, server,
-                               client, test_unit, test_system, server_addr, client_addr, delay, config_path, enable_nfs)
-    sys.exit(status)
+    status: dict = run_pipeline(repository, branch, libos, is_debug, server,
+                                client, test_unit, test_system, server_addr,
+                                client_addr, delay, config_path, enable_nfs)
+    if False in status.values():
+        sys.exit(-1)
```

Comment:
Could you add a print statement on the actual tests that have failed here? This is important for further integration with GitHub Actions.

Reply:
It should print the usual pass and fail statements since they are produced by the wait_and_report function.